universal-login-react/src/ui/commons/Notice.tsx

```diff
@@ -1,14 +1,17 @@
 import React from 'react';
 import './../styles/notice.sass';
-
+import './../styles/themes/Legacy/noticeThemeLegacy.sass';
+import './../styles/themes/Jarvis/noticeThemeJarvis.sass';
+import {useThemeClassFor} from '../utils/classFor';
 export interface NoticeProps {
   message?: string;
 }
 
 export const Notice = ({message}: NoticeProps) => {
+  const theme = useThemeClassFor();
   if (message) {
     return (
-      <div className={'notice-wrapper'}>
+      <div className={`${theme} notice-wrapper`}>
```

Comment:
`useThemeClass()`

Reply:
"React Hook "useThemeClassFor" is called conditionally. React Hooks must be called in the exact same order in every component render" That's why I use
`const theme = useThemeClassFor();`